crates/re_renderer/src/global_bindings.rs

```diff
@@ -33,7 +33,7 @@ pub struct FrameUniformBuffer {
 
     /// How many pixels there are per point.
     /// I.e. the UI zoom factor
-    pub pixels_from_point: f32,
```

Comment:
I couldn't stand the inconsistency of:
* `pixels_per_point`
* `pixels_from_point`
* `pixels_from_points`

Reply:
it was such a nice zoo ;-)